### DISCUSSION TITLE
Digital versus analogue

### DISCUSSION THREAD

**Discussion by craigyjp** [probonopd/MiniDexed#448] [General]
I love the MiniDexed and all the hard work that has gone into development of it but I do miss the hardware side of synthesizer. Here's my latest creation which is a 16 voice bi-timbral analogue Polysynth based on a Juno 60 DCO type synth with an extra bank of oscillators and waveforms and multiple LFOs and EGs plus noise source.

![20230304_202521.jpg](https://user-images.githubusercontent.com/51026305/222924395-36f13ee3-2a0d-4b9c-b0a2-ede5ab51ba75.jpg)

**Comment by probonopd** [comment]
Wow, that is an impressive piece of electronics.
I imagine it has cost half a fortune (or the equivalent time) to put it together.
What I like about doing things in software is that hardware should be widely and cheaply available. Although this is currently not really the case with Raspberry Pis.

**Comment by craigyjp** [comment]
I agree with you, but it's less interesting from my point of view, my
software skills are rather poor so I put a lot of effort into hardware with
my limited skill set. This has kept me busy for about 4-5 weeks in-between
air raid sirens and power outages, things are a little more normal here now
so I can devote more time to the software side of it. It has a Teensy
controller and 6 Picos for the DCOs.

**Comment by Banana71** [comment]
Thank you for letting us be a part of this.  I hope to find a new YouTube video on your channel soon
